sematic/log_reader.py

```diff
@@ -688,6 +703,12 @@ def passes_filter(line: LogLine) -> bool:
         # pointing even if we found nothing in forward direction.
         reverse_cursor_file = original_cursor_file
         reverse_cursor_index = original_cursor_line_index
+    if forward_cursor_file is None and len(lines) == 0 and reverse:
```

Comment:
Same unit test question.

Reply:
yes, covered in test_log_log_lines_reverse